isis/src/base/apps/spiceinit/spiceinit.cpp

```diff
@@ -276,125 +252,31 @@ namespace Isis {
-    }
-    if (ui.WasEntered("EXTRA")) {
-      ui.GetAsString("EXTRA", options.extra);
-    }
-    if (ui.WasEntered("MODEL")) {
-      ui.GetAsString("MODEL", options.model);
-    }
-    if (ui.GetString("SHAPE") == "ELLIPSOID") {
-      options.shape = spiceinitOptions::ELLIPSOID;
+  void getUserEnteredKernel(UserInterface &ui, const QString &param, Kernel &kernel) {
+    if (ui.WasEntered(param)) {
+      kernel = Kernel();
+      // NOTE: This is using GetAsString so that vars like $mgs can be used.
+      vector<QString> kernels;
+      ui.GetAsString(param, kernels);
+      kernel.setKernels(QVector<QString>::fromStdVector(kernels).toList());
     }
-    else if (ui.GetString("SHAPE") == "RINGPLANE") {
-      options.shape = spiceinitOptions::RINGPLANE;
-    }
-    else if (ui.GetString("SHAPE") == "SYSTEM") {
-      options.shape = spiceinitOptions::SYSTEM;
-    }
-    else if (ui.GetString("SHAPE") == "USER") {
-      options.shape = spiceinitOptions::USER;
-    }
-    else {
-      throw IException(IException::Unknown,
-                       "Unknown SHAPE option [" + ui.GetString("SHAPE") + "].",
-                       _FILEINFO_);
-    }
-    options.startpad = ui.GetDouble("STARTPAD");
-    options.endpad = ui.GetDouble("ENDPAD");
-    options.url = ui.GetString("URL");
-    options.port = ui.GetInteger("PORT");
-
-    return options;
   }
 
-
-  /**
-   * Helper function to overwrite the system kernels with specified kernels.
-   *
-   * @param(in/out) kernel The Kernel object to overwrite
-   * @param userKernels The vector of specified kernels
-   */
-  void getUserEnteredKernel(Kernel &kernel,
-                            const std::vector<QString> &userKernels) {
-    if (!userKernels.empty()) {
-      kernel.setKernels(QVector<QString>::fromStdVector(userKernels).toList());
-    }
-  }
-
-
-  /**
-    * Attempt to create a camera model from a set of kernels.
-    *
-    * @param(in/out) icube The Cube to create the camera from. If attach is true
-    *                      in the options, then the SPICE data will be written
-    *                      to the Cube's file.
-    * @param p The process object that the Cube belongs to
-    * @param options The spiceinit options
-    * @param(out) log The Application log
-    * @param lk The leap second kernels
-    * @param pck The planetary constant kernels
-    * @param targetspk The target state kernels
-    * @param ck The camera kernels
-    * @param fk The frame kernels
-    * @param ik The instrument kernels
-    * @param sclk The spacecraft clock kernels
-    * @param spk The spacecraft state kernels
-    * @param iak The instrument addendum kernels
-    * @param dem The digital elevation model
-    * @param exk The extra kernels
-    *
-    * @return If a camera model was successfully created
-   */
-  bool tryKernels(Cube *icube, Process &p,
-                  const spiceinitOptions &options,
-                  Pvl *log,
+  /** 
```

Comment:
Same as above, why remove most of the documentation?

Reply:
There were a lot of changes in there originally that needed to be reverted so it was easier to revert the file and than manually bring things back in. I guess I forgot the documentation strings.